edenia/antelope-tools
# Chore/review cpu benchmark 1188

### Review cpu benchmark

### What does this PR do?

- Resolve #1188
- Add a function for daily clearing of cpu benchmark records from a year ago
- Refactor frontend code to format the data of the cpu benchmarks
- Change milliseconds to microseconds
- Change duplicated icon in the sidebar

### Steps to test

1. Run the project locally with libre testnet configuration
2. Go to cpu benchmark page
3. Check the page

### Screenshot

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/f142a2d0-2996-41f5-bf04-a49243fd7258)
